draft-irtf-cfrg-vdaf.md

```diff
@@ -37,13 +37,13 @@ normative:
   AES:
     title: "Advanced Encryption Standard (AES)"
     author:
-      - ins: M. Dworkin
-      - ins: E. Barker
-      - ins: J. Nechvatal
-      - ins: J. Foti,
-      - ins: L. Bassham
-      - ins: E. Roback
-      - ins: J. Dray Jr.
+      - name: Morris J. Dworkin
+      - name: Elaine Barker
+      - name: James R. Nechvatal
+      - name: James Foti
+      - name: Lawrence E. Bassham
+      - name: E. Roback
+      - name: James F. Dray Jr.
```

Comment:
Maybe we should add the `ins` field here, since the heuristic gets it wrong
```suggestion
      - name: James F. Dray Jr.
        ins: J. Dray Jr.
```

Reply:
Done, and added a comment.